perun-base/src/test/resources/test-schema.sql

```diff
@@ -180,17 +180,18 @@ create table groups (
 );
 
 create table members (
-	id integer not null,
-	user_id integer not null,
-	vo_id integer not null,
+	id integer not null PRIMARY KEY,
```

Comment:
As @stavamichal stated, we define constraints (with exception of not null and default) after table definition, so we can name them. I believe, it has something to do with naming automatically created indexes etc. Since we change schema from time to time, we need to now, ho to explicitly address anything.